ema_workbench/analysis/plotting_util.py

```diff
@@ -671,8 +671,17 @@ def prepare_pairs_data(
     filter_scalar : bool, optional
 
     """
-    if isinstance(outcomes_to_show, str):
-        raise EMAError("for pair wise plotting, more than one outcome needs to be provided")
+    if outcomes_to_show is not None:
+        if not isinstance(outcomes_to_show, list):
+            raise TypeError(
+                f"For pair-wise plotting multiple outcomes need to be provided.\n"
+                f"outcomes_to_show must be a list of strings or None, instead of a {type(outcomes_to_show)}"
+            )
+        # TODO: Apparently an empty list [] is allowed. Is this intended?
+        elif len(outcomes_to_show) == 1:
```

Comment:
@quaquel when I defined `len(outcomes_to_show) < 2`, test failed because the test where inputting an empty list `[]` (with length 0). Apparently this is allowed, is this intended?

Reply:
if you insert None, or an empty list, it defaults to all outcomes in prepare_data. See [prepare_data](https://github.com/quaquel/EMAworkbench/blob/2cfca39a1962bf044ef94d171f44c74d18840bc7/ema_workbench/analysis/plotting_util.py#L704).

Reply:
Right, then we should explicitly allow empty lists.

Reply:
Not sure. I know that a while ago, I changed everything explicitly to None instead of using empty lists. Can't recall exactly why (perhaps because lists are mutable?)